### PR TITLE
Fix manilaclient tests when insecure ssl is used

### PR DESCRIPTION
**Pull request by vuntz**
This depends on https://review.openstack.org/280193 (Fix functional tests to respect insecure SSL option)

### CODE REVIEW

File: features/step_definitions/functional_tests/functional_tests_steps.rb [hunk 1]
```diff
@@ -50,6 +50,8 @@
     admin_password: json_keystone["attributes"]["keystone"]["admin"]["password"],
     admin_tenant_name: json_keystone["attributes"]["keystone"]["admin"]["tenant"],
     admin_auth_url: admin_auth_url.to_s,
+    insecure: json_keystone["attributes"]["keystone"]["ssl"]["insecure"] ||
```
Comment (aplanas) [inline features/step_definitions/functional_tests/functional_tests_steps.rb:53]: Below we are also checking for nova, do we need to include it also here?

Comment (vuntz) [inline features/step_definitions/functional_tests/functional_tests_steps.rb:53]: No, the config we're creating here is only for manilaclient, while the test for nova below is only for novaclient.

Comment (aplanas) [inline features/step_definitions/functional_tests/functional_tests_steps.rb:53]: Oh true, thanks!

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**